CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.1.3
+
+- Fixed escaped double quotes in `@rx` operator
+
```

Comment:
Let's also not bump the version and CHANGELOG in the PR, we will do that as a separate commit once changes are finalized

Reply:
Removed.